test/e2e/persistent_volumes.go

```diff
@@ -66,58 +64,111 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 			persistentVolumeTestCleanup(c, config)
 		}()
 
+		// Create the nfs server pod
 		pod := startVolumeServer(c, config)
 		serverIP := pod.Status.PodIP
 		framework.Logf("NFS server IP address: %v", serverIP)
 
+		// Define the PersistentVolume and PersistentVolumeClaim
 		pv := makePersistentVolume(serverIP)
 		pvc := makePersistentVolumeClaim(ns)
 
-		framework.Logf("Creating PersistentVolume using NFS")
+		// Create the PersistentVolume and wait for PersistentVolume.Status.Phase to be Available
+		framework.Logf("Creating PersistentVolume")
 		pv, err := c.PersistentVolumes().Create(pv)
```

Comment:
I did not notice it earlier, but these `framework.Failf` calls around leave PV in default namespace, right?

Please add something like 
```
defer c.PersistentVolumes().Delete(pv.Name)
```